### ISSUE TITLE
Highcharter doesn't work in updated Rstudio & R.

### ISSUE THREAD

**Issue by jhk0530** [jbkunst/highcharter#619]
I tried this demo code. 

and it doesn't plot in viewer panel in R studio. 

Instead, it takes a lot of CPU usage and crashed.

When I run this code in `RGui` (not R studio) it worked.

So I think this doesn't related with updated version of `R` itself.

```R
highchart() %>% 
  hc_chart(type = "column") %>% 
  hc_title(text = "A highcharter chart") %>% 
  hc_xAxis(categories = 2012:2016) %>% 
  hc_add_series(data = c(3900,  4200,  5700,  8500, 11900),
                name = "Downloads")
```

I'm not sure about whether this problem is caused by `Rstudio` or `Highcharter`.

- `Rstudio` Version `1.2.5042`
- `Highcharter` Version `0.7.0.9001`

**Comment by mtyszler** [comment]
Hi @jhk0530 

I also figured there was an issue with other highcharts I had. 

My conclusion was that the problem is related to `R 4.0.0`, since with the latest R studio and `R 3.6.3` it all works, but not with `R 4.0.0`.

You can see the changes within `R 4.0.0` [here](https://cran.r-project.org/bin/windows/base/NEWS.R-4.0.0.html).

I couldn't track the exact issue, but I suspect has to do with the plot function changes, but I could be wrong

**Comment by mtyszler** [comment]
Hi @jbkunst . Do you know what update in R 4.0.0 are crashing highcharts?

**Comment by jbkunst** [comment]
Hi @jhk0530  @mtyszler 

@jhk0530, what OS do you have? I tested on windows and I replicated having the same behaviour.

**Comment by mtyszler** [comment]
I used in windows 10.

I compared the older R studio to the newest R studio,  both in combination with R 4.0 and R 3.6.

The problem happened only with R 4.0, regardless of the R studio version.

**Comment by jhk0530** [comment]
same I used win 10.

I think this isn't only issue with `highcharter`.

some shiny functions also make `Rstudio` viewer panel to crash.

**Comment by jbkunst** [comment]
Doy you have a reprex for thas shiny functions? I'll open an issue to https://github.com/rstudio/rstudio

**Comment by jhk0530** [comment]
here's one example with my own CRAN package `shinyCyJS`.

```R
install.packages('shinyCyJS')
library(shinyCyJS)
shinyCyJS(list(buildNode('a'), buildNode('b'), buildEdge('a','b')))
```

It should plot graph in viewer panel like below ( this is from `3.6.3` )

![image](https://user-images.githubusercontent.com/6457691/81129081-29504880-8f7e-11ea-8599-343793247060.png)

but it frozen in R 4.0

**Comment by mtyszler** [comment]
Check if https://github.com/rstudio/rstudio/issues/6737 fixes it

**Comment by jhk0530** [comment]
It seems this issues has fixed in `Rstudio` version 1.3. 

I'll close this issue
